common/test_utils.cpp

```diff
@@ -144,4 +159,18 @@ void TestSlog::SendTxn(const Transaction& txn) {
   msg.SendTo(client_socket_);
 }
 
+Transaction TestSlog::RecvTxnResult() {
+  {
```

Comment:
This brace should be removed, and also this function does not always return something. Someone had `std::length_error` because of this reason (https://stackoverflow.com/questions/32410394/terminate-called-after-throwing-an-instance-of-stdlength-error-what-basic). Still, it should not reach the code path where it does not return so not entirely sure that this would fix the problem.

Reply:
Good catch, fixed